pkg/common/yaml_decoder.go

```diff
@@ -49,5 +58,8 @@ func DecodeFile(ctx context.Context, fileData []byte, scheme *runtime.Scheme, cb
 			return err
 		}
 	}
+	if hasError {
```

Comment:
So before we were logging the errors but not returning them, maybe we could fail asap, and return the error when it encounters

Reply:
+1